oss_src/unity/lib/unity_sframe_builder.hpp

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (C) 2015 Dato, Inc.
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the BSD license. See the LICENSE file for details.
+ */
+#ifndef GRAPHLAB_UNITY_SFRAME_BUILDER_HPP
+#define GRAPHLAB_UNITY_SFRAME_BUILDER_HPP
+
+#include <vector>
+#include <set>
+#include <sframe/sframe.hpp>
+#include <boost/circular_buffer.hpp>
+#include <unity/lib/api/unity_sframe_builder_interface.hpp>
+
+typedef boost::circular_buffer<std::vector<graphlab::flexible_type>> row_history_t;
+
+namespace graphlab {
+
+class unity_sframe_builder: public unity_sframe_builder_base {
+ public:
```

Comment:
similar to unity_sarray_builder
doxygen docs. + deletion of copy constructor and copy assignment